doc/uftrace.html

```diff
@@ -266,12 +268,14 @@
 ...         prefix: /usr/local
 ...         libelf: [ .green[on]  ] - more flexible ELF data handling
 ...          libdw: [ .green[on]  ] - DWARF debug info support
-...   libpython2.7: [ .green[on]  ] - python scripting support
+...      libpython: [ .green[on]  ] - python tracing & scripting support
+...      libluajit: [ .green[on]  ] - luajit scripting support
 ...    libncursesw: [ .green[on]  ] - TUI support
 ...   cxa_demangle: [ .green[on]  ] - full demangler support with libstdc++
 ...     perf_event: [ .green[on]  ] - perf (PMU) event support
 ...       schedule: [ .green[on]  ] - scheduler event support
 ...       capstone: [ .green[on]  ] - full dynamic tracing support
+...      libunwind: [ .green[on]  ] - stacktrace support (optional for debugging)
```

Comment:
This is actually not installed by `install-deps.sh` script because it's just for debugging purpose. We didn't want to add a dependency without real purposes. I think we better remove `libunwind` part to reduce confusion.

Reply:
@honggyukim Thanks so much for reviewing with us. :)